CollaboratedHRM/Pages/HomePage.cs

```diff
@@ -0,0 +1,29 @@
+﻿using OpenQA.Selenium;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CollaboratedHRM.Pages
+{
+    public class HomePage
+    {
+
+        private IWebDriver _driver;
+
+        public HomePage(IWebDriver driver)
+        {
+            _driver = driver;
+        }
+        private IWebElement LoginName() 
```

Comment:
Try and name your methods to indicate what they do. In this case, 'GetLoginName()' would be a nice replacement